src/mesh/simplemesh.jl

```diff
@@ -44,3 +44,26 @@ list of points and [`FullTopology`](@ref).
 """
 Base.convert(::Type{<:SimpleMesh}, m::Mesh) =
   SimpleMesh(vertices(m), topology(m))
+
+
+"""
+    isinside(point, mesh)
+
+Tells whether a `point` is inside a surface `mesh`. Must be a surface mesh with only triangles as elements. Points that coincide with vertices of the mesh or lies on the face of a triangle return true.
+"""
+function isinside(testpoint::Point{3,T}, mesh::Mesh{3,T}) where T
+  if !(eltype(mesh) <: Triangle)
+    error("This function only works for surface meshes with triangles as elements.")
+  end
+  ex = testpoint .- extrema(mesh)
+  direction = ex[argmax(norm.(ex))]
+  r = Ray(testpoint, direction*2)
```

Comment:
Can you work on these variable names? Some of them like `ex` are difficult to grasp and others like `direction` are too long. Try to stick to a single code style, and use comments if you need to clarify the meaning of variables. Usually, we prefer short variable names in these small functions.